examples/devicefarm/devicepool.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: devicefarm.aws.upbound.io/v1beta1
+kind: DevicePool
+metadata:
+  annotations:
+    meta.upbound.io/example-id: devicefarm/v1beta1/devicepool
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    name: example
```

Comment:
Ideally, we should be able to use `metadata.name` instead of `spec.forProvider.name` but this is not possible to the following limitation https://github.com/upbound/upjet/issues/142. 
Just crosslinking the issue here, no action is required.